src/main/scala/cromwell/engine/backend/runtimeattributes/AttributeMap.scala

```diff
@@ -12,8 +12,16 @@ object AttributeMap {
   }
 }
 
-case class AttributeMap(attrs: Map[String, Seq[String]]) {
-  def get(key: RuntimeKey): Option[String] = attrs.get(key.key).flatMap(_.headOption)
+trait AttributeMapTrait {
```

Comment:
If this is mainly a naming gripe on something Scott's going to be moving around, I'd agree on waiting for the end product and seeing how it looks then.